script.js

```diff
@@ -0,0 +1,84 @@
+// create empty list [{book:title, author:athorName}]
+// put in localStorage
+// fetch from local storage
+// display to user
+// remove from list and update localStorage
+
+const addBookButton = document.querySelector('#add-book');
+const bookList = document.querySelector('.book-list');
+const libBooks = JSON.parse(localStorage.getItem('libraryBooks')) || [];
+
+// Function to update local storage
+function updateLocalStorage() {
+  localStorage.setItem('libraryBooks', JSON.stringify(libBooks));
+}
+// Grabbing the value of the books and the author when the addBookButton is clicked
+addBookButton.addEventListener('click', (e) => {
+  const bookTitle = document.querySelector('#book-title').value;
+  const authorName = document.querySelector('#author-name').value;
+  const message = document.querySelector('.message');
+
+  if (bookList === '' || authorName === '') {
+    message.innerText = 'Please input Title and Author';
+    e.preventDefault();
+  }
```

Comment:
- [OPTIONAL] Your team has done a great job so far :+1: instead of selecting the button and adding a click event to it, I'll suggest you select the form element, and then add a submit event to the form, this is a better solution. Also, suggest preventing the default form behavior`e.preventDefault();` in the first line of the function because currently, the website refreshes anytime the form is submitted. :+1: :100: :smile:  

```sh
const form = document.querySelector('.form');
form.addEventListener('submit', (e) => {
  e.preventDefault();

...other codes...
})
```

Reply:
@V-Blaze, we thank you for your support and thorough feedback